packages/atoms/src/Chip.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { ReactNode } from 'react';
+import { Text } from './Text';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { TouchableOpacity } from 'react-native';
+import { Box } from './Box';
+
+type ChipProps = {
+    title: string;
+    onSelect: () => void;
+    description?: string;
+    icon: ReactNode;
+    isSelected?: boolean;
+};
+
+type ChipStyleProps = {
+    isSelected: boolean;
+};
+const chipStyles = prepareNativeStyle<ChipStyleProps>((utils, { isSelected }) => ({
+    flexDirection: 'row',
+    justifyContent: 'center',
+    alignItems: 'center',
+    alignSelf: 'flex-start',
```

Comment:
I don't understand what you mean by this comment, could you explain better? 

I used the alignSelf, because without it, the chip component has full width. I don't want to calculate width based on content so this is the cleanest solution I found for the component to be "as wide as the content within".

Reply:
Ok, thanks for explanation, let it be then.